infinoted/infinoted-parameter.h

```diff
@@ -240,6 +244,11 @@ infinoted_parameter_convert_flags(gpointer out,
                                   const GFlagsValue* values,
                                   GError** error);
 
+gboolean
+infinoted_parameter_convert_ip_address(gpointer out,
+                                       gpointer in,
+                                       GError** error);
+
```

Comment:
It appears to me that this is already the case: https://github.com/gobby/libinfinity/pull/15/files#diff-3023f5a98c4128f4ad459d1c20dc74c2R46